src/libs/Authentication.js

```diff
@@ -36,7 +36,6 @@ function Authenticate(parameters) {
         partnerUserSecret: parameters.partnerUserSecret,
         twoFactorAuthCode: parameters.twoFactorAuthCode,
         authToken: parameters.authToken,
-        shouldRetry: false,
```

Comment:
We removed the deprecated API, but we still call Network.post for this. Why don't we update this to use our API?

Also, should we have a mechanism to make sure that some requests aren't retried? It seems like it might be important for these requests, but I'm not totally sure.

Reply:
Regarding retries, I don't think the MainQueue actually "retries" any requests right now, it just queues them for later if they haven't been sent yet. So I don't think we need a mechanism to make sure that some requests aren't retried.

I could be wrong though, I would appreciate a double check on this.

Reply:
Ok cool, so since this is on the Main queue it's not retried and we're all good.